src/commands/settings/togglelevelpings.ts

```diff
@@ -0,0 +1,20 @@
+import { DB, MAINTAINERS } from '@root/config';
+import { SageUser } from '@lib/types/SageUser';
+import { Message } from 'discord.js';
+
+export const description = `Toggles whether or not you will receive notifications from Sage on a level up.`;
+export const aliases = ['levelpings', 'lp'];
+
+export async function run(msg: Message): Promise<Message> {
+	const entry: SageUser = await msg.client.mongo.collection(DB.USERS).findOne({ discordId: msg.author.id });
+
+	if (!entry) {
+		throw `Something went wrong. Please contact ${MAINTAINERS}`;
```

Comment:
Throw a database error here. Error contents from errors thrown in the run function arent shown to the user so this will go to the error log.

Reply:
Wrote this before errors were merged, fixing